src/components/WebexActivityStream/WebexActivityStream.js

```diff
@@ -138,19 +170,34 @@ Greeting.propTypes = {
   personName: PropTypes.string.isRequired,
 };
 
+export function TimeRuler({text}) {
+  return <ListSeparator className="time-ruler" role="listitem" text={text} />;
+}
+
+TimeRuler.propTypes = {
+  text: PropTypes.string.isRequired,
```

Comment:
Remove trailing comma, please
```suggestion
  text: PropTypes.string.isRequired
```